src/resolvers/index.ts

```diff
@@ -37,6 +38,9 @@ export const resolvers = {
   SavedItem: {
     tags: savedItemTags,
     item,
+    __referenceResolver: async (savedItem, context: IContext) => {
```

Comment:
umm I'm not sure if `__referenceResolver` is the right syntax? I believe it is `__resolveReference`?

I'm following these docs: https://www.apollographql.com/docs/federation/api/apollo-subgraph/#__resolvereference

Reply:
thank you for catching this 🙏

Reply:
@kkyeboah do we have tests that should have caught this?

Reply:
@bassrock added a test for the resolver